.github/workflows/pr.yml

```
@@ -23,4 +23,5 @@ jobs:
 
       - run: bun run prepublishOnly
       
-      - run: npx pkg-pr-new publish
\ No newline at end of file
+      - run: npx pkg-pr-new publish
+        working-directory: packages/react-native-device-activity
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)